src/api/l_audio.c

```diff
@@ -150,15 +157,43 @@ static const luaL_Reg lovrAudio[] = {
   { "setCaptureDevice", l_lovrAudioSetCaptureDevice },
   { "getCaptureDevice", l_lovrAudioGetCaptureDevice },
   { "listCaptureDevices", l_lovrAudioListCaptureDevices },
+  { "getSpatializerName", l_lovrAudioGetSpatializerName },
   { NULL, NULL }
 };
 
 int luaopen_lovr_audio(lua_State* L) {
   lua_newtable(L);
   luax_register(L, lovrAudio);
   luax_registertype(L, Source);
-  AudioConfig config[2] = { { .enable = true, .start = true }, { .enable = false, .start = false } };
-  if (lovrAudioInit(config)) {
+
+  const char *spatializer = NULL;
+  int spatializerMaxSourcesHint = AUDIO_SPATIALIZER_MAX_SOURCES_HINT;
+  luax_pushconf(L);
+  lua_getfield(L, -1, "audio");
+  if (lua_istable(L, -1)) {
+    lua_getfield(L, -1, "spatializerMaxSourcesHint");
+    if (lua_type(L, -1) == LUA_TNUMBER)
+      spatializerMaxSourcesHint = lua_tointeger(L, -1);
```

Comment:
Maybe this should use optinteger so there are errors if it's an unexpected type (also please use braces for multiline ifs)